bytes/concat.ts

```diff
@@ -1,18 +1,57 @@
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
 // This module is browser compatible.
 
-/** Concatenate the given arrays into a new Uint8Array.
+/**
+ * Concatenate an array of {@linkcode Uint8Array}s.
  *
+ * @example
  * ```ts
  * import { concat } from "https://deno.land/std@$STD_VERSION/bytes/concat.ts";
+ *
  * const a = new Uint8Array([0, 1, 2]);
  * const b = new Uint8Array([3, 4, 5]);
- * console.log(concat(a, b)); // [0, 1, 2, 3, 4, 5]
+ * concat([a, b]); // Uint8Array(6) [ 0, 1, 2, 3, 4, 5 ]
  * ```
  */
 export function concat(buf: Uint8Array[]): Uint8Array;
+/**
+ * @deprecated (will be removed in 0.209.0) Pass in an array instead of a
+ * spread of arguments.
+ *
+ * Concatenate an array of {@linkcode Uint8Array}s.
+ *
+ * @example
+ * ```ts
+ * import { concat } from "https://deno.land/std@$STD_VERSION/bytes/concat.ts";
+ *
+ * const a = new Uint8Array([0, 1, 2]);
+ * const b = new Uint8Array([3, 4, 5]);
+ * concat(a, b); // Uint8Array(6) [ 0, 1, 2, 3, 4, 5 ]
+ * ```
+ */
 export function concat(...buf: Uint8Array[]): Uint8Array;
+/**
+ * @deprecated (will be removed in 0.209.0) Pass in an array instead of a
+ * spread of arguments.
+ *
+ * Concatenate an array of {@linkcode Uint8Array}s.
+ *
+ * @example
+ * ```ts
+ * import { concat } from "https://deno.land/std@$STD_VERSION/bytes/concat.ts";
+ *
+ * const a = new Uint8Array([0, 1, 2]);
+ * const b = new Uint8Array([3, 4, 5]);
+ * concat(a, b); // Uint8Array(6) [ 0, 1, 2, 3, 4, 5 ]
+ * ```
+ */
```

Comment:
I don't think we need to have this jsdoc. This is for implementatioin signature ([ref](https://www.typescriptlang.org/docs/handbook/2/functions.html#overload-signatures-and-the-implementation-signature)) and that is not visible to the users.

Reply:
Ah, yes, you did say that. Fixed.